### PR TITLE
Update readme.md add web framework

### PR DESCRIPTION
**Pull request by floscodes**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -207,6 +207,7 @@ A general-purpose programming language and toolchain for maintaining robust, opt
 * [zigwasm/wasmer-zig](https://github.com/zigwasm/wasmer-zig) - Bindings for the Wasmer WebAssembly runtime.
 * [zigwasm/wasmtime-zig](https://github.com/zigwasm/wasmtime-zig) - Bindings of Wasmtime.
 * [batiati/mustache-zig](https://github.com/batiati/mustache-zig) - Mustache implementation.
+* [floscodes/zerve](https://github.com/floscodes/zerve) - Simple framework for writing web services and web apps in zig.
```
Comment (catdevnull) [inline readme.md:210]:
```suggestion
* [floscodes/zerve](https://github.com/floscodes/zerve) - Simple framework for writing web services and web apps.
```

Implied :)

**Review by catdevnull — COMMENTED** [review]
thanks for collaborating!

File: readme.md (hunk 1 above)
Comment (floscodes) [reply readme.md:210]: Thank you 🙏

**Comment by floscodes** [comment]
> thanks for collaborating!

Thank you too! Should I change sth else?

**Comment by catdevnull** [comment]
No, this is great, thanks!

**Comment by floscodes** [comment]
> No, this is great, thanks!

Thank you too!

**Pull request merged**